ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml

```diff
@@ -53,3 +53,4 @@
     - Clang_32bit                 # OpenJ9
     - OpenSSL                     # OpenJ9
     - nasm                        # OpenJ9
+    - LangPack
```

Comment:
Can we add a comment as to why this is needed please.

Reply:
@karianna Yes, updated.